jobs/projects.rb

```diff
@@ -125,6 +125,13 @@ module Config
       title: 'Prisoner Offender Search',
       teams: ['syscon', 'sed', 'incentives'],
     },
+    {
+      name: 'prisoner-search',
```

Comment:
The projects name is really `hmpps-prisoner-search`?

Reply:
@mattops Do I still need to do this - I'm confused